Dockerfile

```diff
@@ -1,12 +1,12 @@
-FROM python:3
+FROM python:3-stretch
```

Comment:
Yes - the latest docker image is based on debian buster not stretch, the python version in there is 3.8.2 https://hub.docker.com/_/python. Sadly i had to use the stretch version to ensure i got an OS install of libtiff-dev that worked with the pylibtiff. I discussed this in https://github.com/zooniverse/theia/pull/95#issuecomment-611000661

Interestingly `python:3-stretch` is quite old, re-checking https://hub.docker.com/_/python?tab=tags&page=1&name=stretch it looks like `3.7-stretch` is maintained. I've bumped to that base image. that base image has libtiff5-dev version `4.0.8`
```
dpkg -s libtiff5-dev
Package: libtiff5-dev
Status: install ok installed
Priority: optional
Section: libdevel
Installed-Size: 1032
Maintainer: Laszlo Boszormenyi (GCS) <gcs@debian.org>
Architecture: amd64
Multi-Arch: same
Source: tiff
Version: 4.0.8-2+deb9u4
Replaces: libtiff4-dev (<< 4.0.3-6~), libtiff5-alt-dev (<< 4.0.3-6~)
Provides: libtiff-dev
Depends: libc6-dev | libc-dev, libjbig-dev, libjpeg-dev, liblzma-dev, libtiff5 (= 4.0.8-2+deb9u4), libtiffxx5 (= 4.0.8-2+deb9u4), zlib1g-dev
Conflicts: libtiff4-dev (<< 4.0.3-6~), libtiff5-alt-dev (<< 4.0.3-6~)
Description: Tag Image File Format library (TIFF), development files
 libtiff is a library providing support for the Tag Image File Format
 (TIFF), a widely used format for storing image data.  This package
 includes the development files, static library, and header files.
Homepage: http://libtiff.maptools.org
```

Also and this may surprise you (it did me), folks are using pipenv to install libs for python, this pipfile is built using python 3.7 so it appears we should be syncing the python versions here as well -i'm not wedded to 3.7, sadly it's the only version i could get working without building my own base image https://github.com/zooniverse/theia/blob/44589a2211b06ae3aa13b7706617b5341347bce8/Pipfile#L37